docs/topics/native/native-c-interop.md

```diff
@@ -98,21 +98,30 @@ in case of direct dependencies.
 
 #### Filter headers by globs
 
-It is possible to filter headers by globs. The `headerFilter` property value
-from the `.def` file is treated as a space-separated list of globs. If the
-included header matches any of the globs, then the declarations from this header
-are included into the bindings.
-
-The globs are applied to the header paths relative to the appropriate include
-path elements, e.g. `time.h` or `curl/curl.h`. So if the library is usually
-included with `#include <SomeLibrary/Header.h>`, then it would probably be
-correct to filter headers with
-
-```c
-headerFilter = SomeLibrary/**
-```
-
-If a `headerFilter` is not specified, then all headers are included.
+It is possible to filter headers by globs using filter properties from the `.def` file.
+They are treated as a space-separated list of globs.
+
+* To include declarations from headers, use the `headerFilter` property. If the included header matches any of the globs,
+  the declarations are included in the bindings.
+
+  The globs are applied to the header paths relative to the appropriate include path elements,
+  for example, `time.h` or `curl/curl.h`. So if the library is usually included with `#include <SomeLibrary/Header.h>`,
+  it would probably be correct to filter headers with the following filter:
+    
+  ```c
+  headerFilter = SomeLibrary/**
+  ```
+    
+  If `headerFilter` is not specified, then all headers are included.
```

Comment:
Let's mention that it is recommended to explicitly specify `headerFilter` with the most precise glob possible, so that the generated interop library contains only the necessary declarations. It should help avoid various problems when upgrading Kotlin or environment (e.g. Xcode).